### PR TITLE
Add eslint #79

### PR DESCRIPTION
**Pull request by reenko**
Issue #79

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,32 +17,39 @@
...
     "@ceramicstudio/idx": "0.12.1",
+    "@ceramicstudio/idx-tools": "0.11.0",
     "@ensdomains/ensjs": "2.0.1",
+    "@ethersproject/basex": "^5.4.0",
     "@monaco-editor/react": "^4.1.3",
     "@nodefactory/dappeteer": "^2.0.0",
+    "@vercel/node": "^1.12.1",
     "@web3api/client-js": "0.0.1-prealpha.30",
     "@web3api/ethereum-plugin-js": "0.0.1-prealpha.30",
+    "@web3api/ipfs-plugin-js": "^0.0.1-prealpha.32",
```
Comment (cbrzn) [inline package.json:31]: Let's also upgrade `ethereum plugin` and `client js` packages, so we guarantee consistency between packages

**Review by cbrzn — COMMENTED** [review]
This is awesome!  What do you think about also adding husky so we make sure that the code is formatted on every commit?

**Comment by reenko** [comment]
> add husky

Done

**Comment by reenko** [comment]
@cbrzn could you add me into vercel?

**Review by cbrzn — APPROVED** [review]

**Pull request merged**